action.yml

```diff
@@ -0,0 +1,34 @@
+name: Shuttle deploy
+description: "Deploy to shuttle"
+
+inputs:
+  shuttle-deploy-key:

```

Comment:
```suggestion
  deploy-key:
```
Since this is already in shuttle, maybe we don't need to repeat it again?

Reply:
Yes that is simpler and it's pretty intuitive that it is still the **shuttle** deploy key